lib/ip-checker.js

```diff
@@ -26,12 +25,11 @@ IPChecker.check = ip =>
     let errors = new List();
     const octokit = new Octokit({ auth: global.GH_TOKEN });
     const ghIp = octokit.request('GET /meta');
-    const travisIp = dig(['+short', 'nat.travisci.net']);
     // downloaded from https://www.microsoft.com/en-us/download/details.aspx?id=56519
     const azureIp = fs.promises
       .readFile(`${__dirname}/../ServiceTags_Public.json`)
       .then(JSON.parse);
-    Promise.all([travisIp, ghIp, azureIp])
+    Promise.all([ghIp, azureIp])
```

Comment:
Can you also download the latest version of `ServiceTags_Public.json` so we are up-to-date with the GH IPs?